dependencies.yaml

```diff
@@ -838,6 +838,7 @@ dependencies:
       - output_types: [conda, requirements, pyproject]
         packages:
           - rapids-dask-dependency==25.8.*,>=0.0.0a0
+          - nvidia-ml-py
```

Comment:
Note that cudf requires `pynvml>=12.0.0,<13.0.0a0`. Those versions of `pynvml` are compatible with the latest `nvidia-ml-py`, so this should be good.